api/tests/routes/backoffice/finance_test.py

```diff
@@ -185,6 +187,8 @@ def test_validate_incident(self, authenticated_client, force_debit_note):
             booking__amount=10.10,
             newTotalAmount=0,
         )
+        deposit = finance_models.Deposit.query.filter_by(userId=booking_incident.beneficiary.id).first()
+        first_deposit_value = db.session.query(sa.func.get_deposit_balance(deposit.id, False)).first()[0]
```

Comment:
Suggestion : tu dois a priori obtenir la même chose plus simplement par l'ORM.
De plus, dans ces tests BO, je pense qu'on ne teste pas unitairement la fonction SQL, mais qu'on teste plutôt au niveau objet, donc plutôt `user.wallet_balance` que `get_deposit_balance` : ainsi on vérifie bien que le jeune a été recrédité.
```suggestion
        deposit = booking_incident.beneficiary.deposit
        first_deposit_value = user.wallet_balance
```